react/components/AutocompleteInput/README.md

```diff
@@ -252,3 +252,177 @@ const UsersAutocomplete = () => {
 
 ;<UsersAutocomplete />
 ```
+
+#### Size
+
+### small size bar
```

Comment:
I would put the three inputs in the same example. You don't need to repeat all that logic, IMO.